runtime/jcl/win32/syshelp.c

```diff
@@ -217,8 +217,12 @@ char* getPlatformFileEncoding(JNIEnv *env, char *codepage, int size, int encodin
 	/*[PR 94901] Get info about the current code page, not the system code page (CP_ACP) */
 	if (GetCPInfo(cp, &cpInfo) && cpInfo.MaxCharSize > 1) {
 		if (cp == 936) {
-			if (IsValidCodePage(54936))
-				return "GB18030";
+			J9JavaVM* vm = ((J9VMThread *)env)->javaVM;
+			if (J2SE_VERSION(vm) < J2SE_19) {
+				if (IsValidCodePage(54936)) {
+					return "GB18030";
+				}
+			}
 			return "GBK";
 		} else if (cp == 54936) return "GB18030";
```

Comment:
Is this a code page that can be configured on Windows? It seems this case won't work either.

Reply:
I didn't find a way to configure code page to 54396 on my windows. I can only configure it to 396.